test/unit/path_spec.lua

```diff
@@ -366,134 +366,150 @@ describe('path.c', function()
   end)
 
   describe('vim_FullName', function()
-    local function vim_FullName(filename, buf, len, force)
-      filename = to_cstr(filename)
-      return cimp.vim_FullName(filename, buf, len, force)
+    -- These variables get reset in before_each().
+    local buf
+    local do_expand
+
+    local function vim_FullName(path, buflen)
+      buf = cstr(buflen, '')
+      return cimp.vim_FullName(to_cstr(path), buf, buflen, do_expand)
```

Comment:
Lua has multi-return, so this could return `buf, cimp.vim_...`, then `buf` could stay local. And `do_expand` should just be a parameter, there's no reason the Lua test wrapper for `vim_FullName()` needs the same signature as the C version.